src/main/java/de/adesso/kicker/notification/MatchVerificationRequest/MatchVerificationRequest.java

```diff
@@ -0,0 +1,38 @@
+package de.adesso.kicker.notification.MatchVerificationRequest;
+
+import de.adesso.kicker.match.Match;
+import de.adesso.kicker.notification.Notification;
+import de.adesso.kicker.notification.NotificationType;
+import de.adesso.kicker.user.User;
+import javax.persistence.CascadeType;
+import javax.persistence.ManyToOne;
+
+public class MatchVerificationRequest extends Notification {
+
+    @ManyToOne(targetEntity = Match.class, cascade = CascadeType.ALL)
+    private Match match;
+
+    public MatchVerificationRequest(User sender, User receiver, Match match) {
+
+        super(sender, receiver);
+        super.setType(NotificationType.MATCH_VERIFICATION);
+        this.match = match;
+    }
+
+    public Match getMatch() {
+
+        return match;
+    }
+
+    public void generateMessage() {
```

Comment:
At this tim you were still working on it so i did it like thi for now

Reply:
The message attribute is not unnecessary because it can contain various messages like a decline message, or future message contents that cannot be generated without further attributes